build.psm1

```diff
@@ -569,10 +569,28 @@ Fix steps:
             $ReleaseVersion = $ReleaseTagToUse
         } else {
             $ReleaseVersion = (Get-PSCommitId) -replace '^v'
+            $fileVersion = $ReleaseVersion.Split("-")[0]
+        }
+        if (!$ReleaseVersion) {
+            $ReleaseVersion = "6.0.0"
+            $fileVersion = "6.0.0"
+        }
+
+        Write-Verbose "Release version: $ReleaseVersion" -Verbose
+
+        $pwshPath = $Options.Output
+        if ($Environment.IsWindows) {
```

Comment:
The whole `RCEdit` work is in `if ($Environment.IsWindows) {` block (see line 561), so no need to split the code path here.

Reply:
Yeah, forgot rcedit only applied to Windows, will fix